homeassistant/components/starlink/translations/en.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Regenerate this file with `python3 -m script.translations develop`

Reply:
Is there somewhere in the documentation these strings are explained? I didn't even realize they were generated 😅

Reply:
https://developers.home-assistant.io/blog/2020/04/10/translation-handling

Reply:
https://github.com/home-assistant/architecture/blob/master/adr/0009-translations-2.0.md